### ISSUE TITLE
Can't change the population settings

### ISSUE THREAD

**Issue by TedTed** [arx-deidentifier/arx#87]
Hi! In the "Analyze risk" tab, on the bottom of the UI, tab "Population", I can't un-tick the "Use this population" box, nor un-select USA to select another country. When clicking the box, the tick doesn't go away. I reproduced this with the example.deid project, and with some dummy data from generatedata.com.

I'm using ARX 3.5.1 on Linux, running java 1.8.0_112. uname -a returns:
     Linux [redacted] 3.13.0-101-generic #148-Ubuntu SMP Thu Oct 20 22:08:32 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Comment by prasser** [comment]
Dear TedTed,

thanks for your interest in ARX and thanks for reporting this bug!

As far as I can tell, the first step in fixing this issue is to remove the tabs entirely. As you already pointed out, they are currently not working as expected. Unfortunately, they cannot be fixed easily in a manner that makes their function intuitive for our users. The best solution would be to:

(a) Move the tabs "Population uniques" from the bottom of the perspective to the top

(b) Introduce a new tab at the bottom which implements what the tabs discussed here were originally supposed to do: allow the user analyze uniqueness for a selected population. However, the graphs in "Population uniques" already display results for a wide range of sampling fractions, so both views can easily be combined with each other by drawing a vertical line in the graphs at the sampling fraction corresponding to the selected population.

I will put this on our roadmap for ARX 3.6.0, marked the issue as "enhancement" and I will leave it open for now.

Best regards
Fabian

**Comment by TedTed** [comment]
Agreed, thanks for the fast answer and for looking at this! I think your proposed solution is a good idea. Two more comments on the same topic:

- I'm not convinced by the necessity of plotting the uniqueness estimate for all three methods — why not display exclusively Dankar et al.'s decision rule and hide its complexity to the user?
- I think there should be a visible warning that these methods only return good estimates if the dataset is a uniform sample of the population — this probably isn't verified in many cases, and looking at the "Population" tab before actually reading the uniqueness estimation papers, I thought this was taking into account demographic info on the various countries (and so that a uniform sample wasn't necessary).

Best!

**Comment by prasser** [comment]
Just a quick heads up on the requirement to use a uniform sample. I have added this to the in-app documentation for the upcoming version 3.6.0.
